_posts/2018-08-TBD-release-1.5.3.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "Apache Flink 1.5.3 Released"
+date:   2018-08-TBD 12:00:00
+categories: news
+---
+
+The Apache Flink community released the third bugfix version of the Apache Flink 1.5 series.
+
+This release includes more than 20 fixes and minor improvements for Flink 1.5.1. The list below includes a detailed list of all fixes.
```

Comment:
"1.5.1" Should it be changed to "1.5.2"?

Reply:
will change to 1.5.3